src/apps/chifra/pkg/pinlib/download_test.go

```diff
@@ -13,13 +13,16 @@ import (
 
 var manifestJSONSource = `
 {
-  "fileName": "pin-manifest.json",
+  "version": "2",
+  "chain": "mainnet",
+  "chainId": 1,
   "indexFormat": "Qmart6XP9XjL43p72PGR93QKytbK8jWWcMguhFgxATTya2",
   "bloomFormat": "QmNhPk39DUFoEdhUmtGARqiFECUHeghyeryxZM9kyRxzHD",
   "commitHash": "f29699d3281e41cb011ddfbe50b7f01bfe5e3c53",
-  "prevHash": "QmP4i6ihnVrj8Tx7cTFw4aY6ungpaPYxDJEZ7Vg1RSNSdm",
-  "newBlockRange": "000000000-000350277",
-  "newPins": [
+  "names": "QmP4i6ihnVrj8Tx7cTFw4aY6ungpaPYxDJEZ7Vg1RSNSdm",
```

Comment:
We currently make local changes to this file, but I want to change this so there's a 'cononical' file (this one) and an 'edited' file. We can do that once we move over to All Go Code for names.

In the meantime, since there's no real way to edit this file (other than by hand), we can just download it directly.

There's two files. One is a binary cache of the names.tab file. The other is the names.tab file itself. If names.tab is newer, names.bin gets created.

Upshot: We should store the `.tab` file on IPFS. When we download it (if the blockNumber's timestamp is later than the local file), and touch it. Then the .bin will get updated.

We can discuss.